api/lib/application/sessions/index.js

```diff
@@ -27,7 +28,7 @@ exports.register = async (server) => {
       config: {
         validate: {
           params: Joi.object({
-            id: Joi.number().required()
+            id: Joi.number().max(settings.maxId).required()
```

Comment:
Il commence à devenir long ce validateur d'`id`, on pourrait peut-être le factoriser non ? D'autant qu'il a en fait vocation à s'appliquer partout où il y a un `id` dans l'API, on ne voit pas trop pourquoi ça serait restreint aux routes de `session` 🤔 

```suggestion
            id: settings.idValidator
```

Reply:
Done